specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/mfe.json

```diff
@@ -21156,6 +22197,228 @@
       },
       "additionalProperties": false
     },
+    "ServerlessEndpoint": {
+      "additionalProperties": false,
+      "properties": {
+        "authMode": {
+          "$ref": "#/definitions/ServerlessInferenceEndpointAuthMode",
+          "description": "[Required] Specifies the authentication mode for the Serverless endpoint."
+        },
+        "contentSafety": {
+          "$ref": "#/definitions/ContentSafety",
+          "description": "Specifies the content safety options. If omitted, the default content safety settings will be configured",
+          "x-nullable": true
+        },
+        "endpointState": {
+          "$ref": "#/definitions/ServerlessEndpointState",
+          "description": "The current state of the ServerlessEndpoint.",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "inferenceEndpoint": {
+          "$ref": "#/definitions/ServerlessInferenceEndpoint",
+          "description": "The inference uri to target when making requests against the serverless endpoint",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ],
+          "x-nullable": true
+        },
+        "marketplaceSubscriptionId": {
+          "description": "The MarketplaceSubscription ARM ID associated to this ServerlessEndpoint.",
```

Comment:
>MarketplaceSubscription [](http://example.com/codeflow?start=30&length=23)

Marketplace Azure subscription ID? Or is this an Azure resource ID? Should try to clarify this.

Reply:
this is meant to be the resource id for the new /marketplaceSubscriptions resource added by this PR. a user creates a new /marketplaceSubscription resource and then supplies its Azure ID - ie `/subscriptions/<foo>/resourceGroups/<bar>/providers/Microsoft.MachineLearningServices/workspaces/<baz>/marketplaceSubscriptions/coolSub`

which wordage best describes this intetion?